index.html

```diff
@@ -0,0 +1,249 @@
+                        </svg>                        
+                </span>
+                <span>
+                    <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M19.7665 0.369765C19.4331 -0.0289051 18.8665 -0.128573 18.4331 0.17043L16.4998 1.49933C15.5998 0.535877 14.3331 0.00431735 12.9665 0.00431735C10.2665 0.00431735 8.03313 2.197 8.03313 4.92125C8.03313 5.31992 8.06646 5.68537 8.16646 6.05081C5.83313 5.55248 4.43313 3.85813 3.8998 0.801657C3.83313 0.402987 3.53313 0.103985 3.13313 0.00431735C2.76646 -0.0953502 2.36646 0.0707624 2.13313 0.369765C2.06646 0.43621 -3.43354 8.07739 4.53313 14.5558C3.76646 16.1837 1.46646 17.579 0.566463 18.0441C0.13313 18.2435 -0.06687 18.7086 0.03313 19.1737C0.13313 19.6388 0.53313 19.9378 0.999797 19.9378H2.73313C12.8665 19.9378 17.4665 14.9212 17.5665 3.69202L19.6665 1.69867C20.0665 1.36644 20.0998 0.768435 19.7665 0.369765Z" fill="#DBD8D7"/>
+                        </svg>                        
+                </span>
+                <span>
+                    <svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" fill="#DBD8D7"/>
+                        </svg>                        
+                </span>
+            </section>
+            <img src="./assets/illustration-header-mobile.png" alt="illustration header mobile" class="mobile-header">
+        </div>
+        <div class="works">
+            <h1>My Recent Works</h1>
+            <hr>
+            <img src="./assets/img_placeholder.png" alt="recent work image placeholder">
+            <h2>Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <section class="tags">
+                <span class="tag">css</span>
+                <span class="tag">html</span>
+                <span class="tag">bootstrap</span>
+                <span class="tag">Ruby</span>
+            </section>
+            <button type="button">See Project</button>
```

Comment:
you need to add the hover, active and disabled states to the button
![image](https://user-images.githubusercontent.com/42470159/143785857-32fcd2c9-3242-4f54-9a3f-286b8e9ace21.png)